### PR TITLE
ttH 13.6 TeV WpWpJJ_EWK-QCD bkg samples

### PR DESCRIPTION
**Pull request by mobassirameen**
Dear expert,
We have produced one MadGraph cards for background samples at 13.6 TeV for RunIII production, which include:
WpWpJJ_EWK-QCD
Only the beam energy is changed in the run_card.
Please review them, if there are any issues, please inform us.

Mohammad Mobassir Ameen
(ttH+HH MC contact)

adding Adding @hftsoi @attikis @mlizzo

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WpWpJJ_EWK-QCD/WWjj_SS_ewk_qcd_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+set max_npoint_for_channel 0
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm-ckm_no_b_mass
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WpWpJJ_EWK-QCD/WWjj_SS_ewk_qcd_proc_card.dat:34]: please remove those lines

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WpWpJJ_EWK-QCD/WWjj_SS_ewk_qcd_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+  0.0	=  polbeam1  ! beam polarization for beam 1
+  0.0	=  polbeam2  ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+  lhapdf    = pdlabel     ! PDF set                                  
+  320900 = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WpWpJJ_EWK-QCD/WWjj_SS_ewk_qcd_run_card.dat:52]: change to "$DEFAULT_PDF_SETS = lhaid"

Comment (mobassirameen) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WpWpJJ_EWK-QCD/WWjj_SS_ewk_qcd_run_card.dat]: Hi, @menglu21  
I have modified the cards according to your suggestions. Please check.

**Comment by menglu21** [comment]
Hi @mobassirameen thanks, could you please move the cards to 13p6TeV/VBS for better organization

**Comment by mobassirameen** [comment]
Hi @menglu21, I moved to requested directory. please check. thanks

**Comment by menglu21** [comment]
Hi @mobassirameen it seems now you have the cards in two places, could you remove the initial one.

**Comment by mobassirameen** [comment]
Hi @menglu21 , It is removed. Please check!
Thanks

**Pull request merged**